azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/style/ChatCompositeTheme.kt

```diff
@@ -38,6 +43,14 @@ internal fun ChatCompositeTheme(
         unreadMessagesIndicator = RoundedCornerShape(100.dp)
     )
 
+    val isDarkTheme = isSystemInDarkTheme()
+    val view = LocalView.current
+
+    SideEffect {
```

Comment:
I don't think we should do this.

It is escaping the bounds of the component/view.